### ISSUE TITLE
[Bug] 

### ISSUE THREAD

**Issue by s544838371** [lobehub/lobe-commit#5]
### 💻 系统环境 | Operating System

Windows

### 🌐 CLI 版本 | CLI Version

最新版

### 🐛 问题描述 | Bug Description

输入后无法进行任何设置，包括exit回车都不管用
```
lobe-commit --config
```

### 📷 复现步骤 | Recurrence Steps

npm install -g @lobehub/commit-cli
cd D:\Git项目\lobe
lobe-commit --config

**Comment by s544838371** [comment]
> 🎉 This issue has been resolved in version 2.4.0 🎉
> 
> The release is available on [npm package (@latest dist-tag)](https://www.npmjs.com/package/@lobehub/commit-cli/v/2.4.0)
> 
> Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot 📦🚀

@canisminor1990 
# 更新后使用lobe-commit --config出现问题
```
PS D:\Git项目\lobe> lobe-commit --config
file:///C:/Users/***/AppData/Roaming/npm/node_modules/@lobehub/commit-cli/dist/cli.js:2
#!/usr/bin/env node
^

SyntaxError: Invalid or unexpected token
    at ESMLoader.moduleStrategy (node:internal/modules/esm/translators:119:18)
    at ESMLoader.moduleProvider (node:internal/modules/esm/loader:468:14)
    at async link (node:internal/modules/esm/module_job:68:21)

Node.js v18.16.1
NativeCommandExitException: Program "node.exe" ended with non-zero exit code: 1.
PS D:\Git项目\lobe>
```

**Comment by canisminor1990** [comment]
> > 🎉 This issue has been resolved in version 2.4.0 🎉
> > 
> > The release is available on [npm package (@latest dist-tag)](https://www.npmjs.com/package/@lobehub/commit-cli/v/2.4.0)
> > 
> > Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot 📦🚀
> 
> @canisminor1990 
> # 更新后使用lobe-commit --config以及其他命令均出现如下问题
> 错误提示如下：
> ```
> PS D:\Git项目\lobe> lobe-commit --config
> file:///C:/Users/***/AppData/Roaming/npm/node_modules/@lobehub/commit-cli/dist/cli.js:2
> #!/usr/bin/env node
> ^
> 
> SyntaxError: Invalid or unexpected token
>     at ESMLoader.moduleStrategy (node:internal/modules/esm/translators:119:18)
>     at ESMLoader.moduleProvider (node:internal/modules/esm/loader:468:14)
>     at async link (node:internal/modules/esm/module_job:68:21)
> 
> Node.js v18.16.1
> NativeCommandExitException: Program "node.exe" ended with non-zero exit code: 1.
> PS D:\Git项目\lobe>
> ```

现在呢？

**Comment by s544838371** [comment]
目前在家里的电脑装了下，上午提的错误提示已经解决，可以正常使用了，等明天上午再试一下另外一台电脑
